### ISSUE TITLE
Modify Restaurant results alt tag

### ISSUE THREAD

**Issue by julia-codes** [msjuliaalice/mws-restaurant-stage-1#18]
main.js
153  image.alt = restaurant.name;

Good work, you have added alt tag, but it should have descriptive alt text that succinctly describes what the image is.
